io.github.syntalos.yaml

```diff
@@ -0,0 +1,228 @@
+id: io.github.syntalos
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-23.08
+separate-locales: false
+
+command: syntalos
+rename-icon: syntalos
+rename-desktop-file: syntalos.desktop
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --filesystem=host
+  - --socket=system-bus
+  - --socket=session-bus
+  - --allow=devel
+  - --allow=bluetooth
+  - --env=PATH=/app/bin:/usr/bin:/usr/sbin
+  - --env=GST_PLUGIN_SYSTEM_PATH=/app/lib/gstreamer-1.0:/usr/lib/extensions/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0
```

Comment:
The last path hardcodes the arch triplet in it and the second one should not be needed. No extensions are mounted there.

Reply:
You can set this via a wrapper for the main executable since that'd deal with runtime changes automatically.


finish-args can't do variable substitutions.

Reply:
> I hate wrappers... But maybe we can avoid one still, I have to test the result.

The other option is to just put the aarch64 libdir in there too since one will not have any effect on the other.